src/index.ts

```diff
@@ -1,9 +1,12 @@
 import { Computed, Context, Dict, h, Logger, omit, Quester, Session, SessionError, trimSlash } from 'koishi'
 import { Config, modelMap, models, orientMap, parseInput, sampler, upscalers } from './config'
 import { ImageData, StableDiffusionWebUI } from './types'
-import { closestMultiple, download, forceDataPrefix, getImageSize, login, NetworkError, project, resizeInput, Size } from './utils'
-import {} from '@koishijs/translator'
-import {} from '@koishijs/plugin-help'
+import {
+  closestMultiple, download, forceDataPrefix, getCkptList, getEmbeddingsList, getHypernetworksList, getImageSize, getLoraList, loadModelPreviewImage,
+  login, NetworkError, project, readModelInfo, resizeInput, Size,
+} from './utils'
+import { } from '@koishijs/translator'
+import { } from '@koishijs/plugin-help'
```

Comment:
This is an unneccessary change.

Reply:
I used eslint config from @koishijs/eslint-config

Reply:
Yes, but do it in the new PR, generally we should do one thing in a single PR.